Dvorak-Right-Handed/KeyboardView.swift

```diff
@@ -125,16 +125,16 @@ class KeyboardView: UIView {
         ▿ Optional<String>
           - some : "A10"
         */
-        
-        delete = !delete
-        if delete {
+
+        if deleteTimer == nil {
             deleteTimer = Timer.scheduledTimer(withTimeInterval: 0.1, repeats: true, block: { timer in
                 self.kvc?.textDocumentProxy.deleteBackward()
             })
-
-        } else {
-            //do I need to invalidate timer anywhere else?
+        }
+        
+        if sender.state == .ended {
             deleteTimer?.invalidate()
+            deleteTimer = nil
```

Comment:
I was going to suggest not deleting the timer but after looking up documentation it appears the timer can't be reused once stopped so it seems the OS designers indented for these timers to be discarded after use. So long story short LGTM.